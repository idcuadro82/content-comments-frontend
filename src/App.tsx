import React, { FunctionComponent } from 'react';
import {
  BrowserRouter as Router,
  Route,
  Redirect,
  Switch
} from 'react-router-dom';
import Book from './components/Book/Book';
import './App.css';

const App: FunctionComponent = () => {
  return (
    <Router>
      <Switch>
        <Route path="/book1">
          <Book volume="1" />
        </Route>
        <Redirect exact from="/" to="/book1" />
      </Switch>
    </Router>
  );
};

export default App;
