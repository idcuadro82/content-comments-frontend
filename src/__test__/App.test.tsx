import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

describe('<App>', () => {

  it('Render <APP>', () => {
    const { container } = render(<App />);
    expect(container).toMatchSnapshot();
  });
});
