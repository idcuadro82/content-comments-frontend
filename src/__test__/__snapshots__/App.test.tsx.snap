// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App> Render <APP> 1`] = `
<div>
  <div
    class="document"
  >
    <div
      class="document-content"
    >
      Book 
      1
    </div>
  </div>
</div>
`;
